html/index.php

```diff
@@ -99,13 +116,29 @@ function getUserReplyText($post_id) {
         <div class="card-body">
           <p class="card-title"><b><?= "{$t['id']}" ?></b> <?= "{$t['name']}" ?> <small><?= "{$t['updated_at']}" ?></small></p>
           <p class="card-text"><?= "{$t['text']}" ?></p>
+          
           <!--返信課題はここから修正しましょう。-->
           <?php if (isset($t['reply_id'])) { ?>
-            <a href="index.php?reply=<?= "{$t['id']}" ?>">[返信する]</a> <a href="/view.php?id=<?= "{$t['reply_id']}" ?>">[返信元のメッセージ]</a>
+            <p><a href="index.php?reply=<?= "{$t['id']}" ?>">[返信する]</a>  <a href="/view.php?id=<?= "{$t['reply_id']}" ?>">[返信元のメッセージ]</a></p>
           <?php } else { ?>
             <p><a href="index.php?reply=<?= "{$t['id']}" ?>">[返信する]</a></p>
           <?php } ?>
           <!--返信課題はここまで修正しましょう。-->
+          <form method="POST" name="like_form">
+            <input type="hidden" name="post_id" value="<?= "{$t['id']}" ?>"/>
+          <!--input type="submit" name="送信"/ -->
+          <?php if (hasFavorite($t['id'],$_SESSION['user_id']) ==0) { ?>
+          <!-- 1.いいねボタンを押す　2.name=like_formのformが実行される 3.index.phpが再読み込みされる（なぜならばformにactionが指定されていないから-->
+            <input type="hidden" name="nice_button"/>
+            <a href="#" onclick="this.parentNode.submit()"><img class="favorite-image" src='/images/heart-solid-gray.svg'></a>
+          <?php } else { ?>
+            <input type="hidden" name="delete_button"/>
+            <a href="#" onclick="this.parentNode.submit()"><img class="favorite-image" src='/images/heart-solid-red.svg'></a>
+          <?php } ?>
+          <?php echo getFavoriteUsers($t['id']) ?> <!-- 下にいいねの数値を取る todo -->
```

Comment:
[must]

todo対応してるのでこのコメントは不要かと
消しましょう

```suggestion
          <?php echo getFavoriteUsers($t['id']) ?>
```